src/MainnetController.sol

```diff
@@ -253,6 +261,51 @@ contract MainnetController is AccessControl {
         );
     }
 
+    /**********************************************************************************************/
+    /*** Relayer Aave functions                                                                 ***/
+    /**********************************************************************************************/
+
+    function depositAave(address atoken, uint256 amount)
+        external
+        onlyRole(RELAYER)
+        isActive
+        rateLimited(
+            RateLimitHelpers.makeAssetKey(LIMIT_AAVE_DEPOSIT, atoken),
+            amount
+        )
+    {
+        IERC20 underlying = IERC20(IAToken(atoken).UNDERLYING_ASSET_ADDRESS());
+        IAavePool pool    = IAavePool(IATokenWithPool(atoken).POOL());
+
+        // Approve underlying to Aave pool from the proxy (assumes the proxy has enough underlying).
+        proxy.doCall(
+            address(underlying),
+            abi.encodeCall(underlying.approve, (address(pool), amount))
+        );
+
+        // Deposit underlying into Aave pool, proxy receives atokens
+        proxy.doCall(
+            address(pool),
+            abi.encodeCall(pool.supply, (address(underlying), amount, address(proxy), 0))
+        );
+    }
+
+    function withdrawAave(address atoken, uint256 amount)
+        external onlyRole(RELAYER) isActive returns (uint256 amountWithdrawn)
+    {
+        IAavePool pool = IAavePool(IATokenWithPool(atoken).POOL());
+
+        // Withdraw underlying from Aave pool, decode resulting amount withdrawn.
+        // Assumes proxy has adequate atokens.
+        amountWithdrawn = abi.decode(
+            proxy.doCall(
+                address(pool),
+                abi.encodeCall(pool.withdraw, (IAToken(atoken).UNDERLYING_ASSET_ADDRESS(), amount, address(proxy)))
+            ),
+            (uint256)
+        );
```

Comment:
Small formatting change to get under line limit
```suggestion
        amountWithdrawn = abi.decode(
            proxy.doCall(
                address(pool),
                abi.encodeCall(
                    pool.withdraw,
                    (IAToken(atoken).UNDERLYING_ASSET_ADDRESS(), amount, address(proxy))
                )
            ),
            (uint256)
        );
```

Reply:
changed